engines/director/detection_tables.h

```diff
@@ -419,6 +419,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "shrmess",			"Schoolhouse Rock!: 1st-4th Grade Math Essentials" },
 	{ "simpsonsplayer",		"The Simpsons Cartoon Player" },
 	{ "simpsonsstudio",		"The Simpsons Cartoon Studio" },
+	{ "singingbirdbook",		"Jan Lindblad presenterar den sjungande Fågelboken" },
```

Comment:
If this is swedish-only it should be listed among the swedish titles (line 1395)